app/controllers/hub/users_controller.rb

```diff
@@ -28,6 +28,14 @@ def update
       end
     end
 
+    def resend_invitation
+      user = User.find_by(id: params[:user_id])
```

Comment:
I'd recommend narrowing this down to a subset of users that should be accessible to the current user, and using `find(params[:id])` to raise a not found error if the user id is not within the set of accessible users.

Reply:
What are the access expectations for this action? I'm not sure who can currently see the user edit page, aside from admins.

Reply:
There is also currently a resend user invitation endpoint in users/invitations controller that does very similar logic -- I think we should consolidate to using one or the other!

Reply:
Let me double check.. If that is the case, I'm guessing you're correct in that it's only admins.